### ISSUE TITLE
Cannot put gmail account to sleep

### ISSUE THREAD

**Issue by CheechGe** [wavebox/waveboxapp#1080]
* Wavebox Version: [e.g. 3.0.0]
* Operating System & Version: [e.g. Windows 10, Mac OSX 10.3.4, Ubuntu 17.10]
* (Linux only) Install method: [e.g. Debian installer, Snap Package, Tar, Apt-Repo, etc...]
* Account type (if applicable): [e.g. Gmail, Office 365, Trello, etc...]

**Expected & actual behavior**
Some information about what you expect to happen and what actually happens.

**Steps to reproduce**
Are there any steps that we can take to reproduce the bug?

**Is the bug persistent or intermittent?**
Do you only see it sometimes or does it always happen?

**Screenshots**
Screenshots often help us narrow down a bug more easily.

**Additional information**

**Comment by nmat** [comment]
Can you provide information on how you have setup the account in wavebox regarding auto sleep and so on?

You will find the preferences if you right click the account and go to: Account Settings

**Comment by CheechGe** [comment]
Hello

There are 9 sevices for this account as shown below.  There was only
parameter I could find related to Sleep is shown in the ss below and all
the services had the same setting.

As I mentioned I have a 2nd PC with Wavebox on it and I can put the primary
gmail account to sleep.  Both PCs have W10 but the one that works has the
Home Edition and the other has Home Pro edition.  Wavebox is setup to sync
these two versions.  The Sleep setting on the working version is the same.
The not working gmail was the first service set up when WB was first
installed over 1 year ago and has has not been touch execept for maybe a
icon color change.

Let me know if there is a specific setting you need me to check as there
are a lot and I don't know which might effect this situation.  Besides, the
I would think becuase WB on the second PC was setup just recently and was
created by synching from the origianl version that all the setting would be
the same for all the services.  If thru that would mean the the problem is
isolated the something being different on the first PC.


Regards,


F
[image: image.png]




[image: image.png]

**Comment by Thomas101** [comment]
Thanks for reporting. Do you have any other Wavebox windows open (e.g. from opening a link, or composing in a new Window)? Sometimes the app can refuse to sleep an account if there is a child window open.

**Comment by CheechGe** [comment]
No, nothing else was open at the time.  I started from scratch by rebooting
my PC and the first thing I tested was this situation.

**Comment by CheechGe** [comment]
Hi Thomas:

Well, this is strange.  I rebooted my system again after a couple of days
and this time I was able to put the gmail account to sleep.  This started
me thinking about what might have been different or had happened before the
problem started occurring, and then I remembered something.  There was a
link in an email and I attempted to display the link in the WB browser
rather than my default browser as I do much of the time, but for some
reason the page would not come up.  I don't recall getting any error
message but just the spinning circle, so being my impatient self I closed
the window as it wasn't important.  I don't believe I had to force the
window closed.

I just continued on not thinking much of it until the sleep issue happen
which might have been several days afterwards.  I know I both closed WB and
also did a reboot to see if the problem would correct itself which is what
I would have expected if something didn't clean up correctly.  I have no
explanation why the problem was still there after the first reboot but
disappeared after the second reboot.  Anyway, a little more info for you.

Regards,

Frank

**Comment by Thomas101** [comment]
Thanks very much Frank. I'll have a look over the code our end to see if there is anything that stands out. If this comes up again, it would be great to know about it and anything else notable in the run-up to the failure. It might give us some more clues as to what caused it.
